static/js/components/CourseList.js

```diff
@@ -119,7 +120,7 @@ class CourseList extends React.Component {
             <br/>
             <p>You need to pass {totalCourses - coursesPassed} more courses
               before you can apply for the <strong>{program.title}</strong> Master’s Degree.</p>
-            <MuiThemeProvider>
```

Comment:
ah, indeed we don't! if I remove it things look the same as well (I believe this change was introduced to fix a styling issue).

without:
![without](https://cloud.githubusercontent.com/assets/6207644/15828411/3294d25c-2bdd-11e6-9fca-6e9879bcacc8.png)

with:
![with](https://cloud.githubusercontent.com/assets/6207644/15828414/3739072e-2bdd-11e6-9734-d4fb9a2f8a6c.png)